### DISCUSSION TITLE
How to access entity findMany() where entity is a reserved word

### DISCUSSION THREAD

**Discussion by marceloverdijk** [prisma/prisma#12498] [Q&A] [answered]
I have a `Constructor` prisma model which generates the following in the Prisma client:

     * // Get all Constructors
     * const constructors = await prisma.constructor.findMany()

However, when calling it with `prisma.constructor.findMany()` I get:

`ctx.prisma.constructor.findMany is not a function`

which makes probably sense ;-)

Is there a workaround to access the `constructor` path to execute a query on it?

**Comment by austincrim** [comment]
Hey @marceloverdijk 👋 ,

I don't think it is possible to work around this, it's a constraint of JavaScript itself.

The only option is to rename your model to something that's not a reserved keyword 😄.

**Reply by marceloverdijk** [reply]
I was hoping for something like `prisma['Constructor'].findMany()` but had no luck yet.

**Reply by marceloverdijk** [reply]
@austincrim I confused you? To avoid `prisma.constructor.findMany()` not working because of `constructor`, I was hoping there is another way to access the `Constructor` table, like dynamically by it's name from a map or so... I know a bit nasty but at least that way I could use my `Constructor` model. 
`Constructor` (with capital `C`) is fine I think, it only fails because it uses `constructor` instead in the generated Prisma client.

**Reply by marceloverdijk** [reply]
E.g. with Nexus you can do:

```
type Props = {
  constructors: NexusGenFieldTypes['Constructor'][]
}
```

**Comment by marceloverdijk** [comment, marked as answer]
Actually it is possible ;-)

`const constructors = await prisma['Constructor'].findMany()`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
